src/main/java/seedu/address/model/person/Name.java

```diff
@@ -13,11 +13,10 @@ public class Name {
             + "and it should include given name and surname";
 
     /*
-     * The first character of the address must not be a whitespace, otherwise " " (a
-     * blank string) becomes a valid input. The name must contain at least two
-     * words, for first and last name.
+     * The name must only contain alphanumeric characters and spaces.
+     * It must contain at least two words, for given name and surname.
      */
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}]+ [\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}]+ [\\p{Alnum} ]*";
```

Comment:
P.S. I went to experiment with this regex thingy, seems like this works fine as well? Not 100% certain if another `[\\p{Alnum}]` is necessary or redundant, open to feedback!

Reply:
The reason for the extra `[\\p{Alnum}]` is to enforce the "at least two word" rule. Without the extra portion, `Tan ` is an acceptable name.

Reply:
Okay! I tested it, and it seems to work the same, so your edit seems good! Another thing is, do we want the name to be Alphanumerical, or just alphabetical?

Reply:
@k-hian 
This is with her edited version:
![Screenshot 2024-10-23 161704](https://github.com/user-attachments/assets/1d099b1d-ed65-411c-b845-8d8f44b68787)

Reply:
Oh wait I think I see the issue here! Cassandra's tweak allows double spacing between the first name and surname. 

So the real issue is that the version without the additional regex accepts
`n/Jane  Doe` as a valid input, whereas the longer version does not allow the double spacing.

Actual version: 
![Screenshot 2024-10-23 162940](https://github.com/user-attachments/assets/2434e8a1-b0b6-4f5c-8d48-509dfbf3d389)
Proposed edit:
![Screenshot 2024-10-23 162855](https://github.com/user-attachments/assets/df942436-34b4-4881-af48-d59febcefb50)

Reply:
@k-hian As in, only the previous line with 3 `[\\p{Alnum}]`s can enforce correctly? Or the current one with 2 can as well XD

Reply:
@sannie-beep ooh good point there, I'm actually fine with alphabetical! Can't seem to see a need for numbers.. may need others' inputs to decide as well~

Reply:
I think the reason it works is because the name is trimmed by the ParserUtil. I guess in this case, function-wise it doesn't matter as much which regex validation we use, but on a developer-as-maintainer standpoint, I think it might be useful to leave the extra portion in to indicate the name input requirement.